contrib/scripts/dumpcerts.sh

```diff
@@ -42,6 +42,18 @@ set -o nounset
 
 USAGE="$(basename "$0") <path to acme> <destination cert directory>"
 
+# Platform variations
+case `uname` in
```

Comment:
Could you please use `$()` instead of backticks, just to follow [best practices](http://mywiki.wooledge.org/BashFAQ/082)?

Reply:
Done. If you want me to squash my commits, just tell me.

Reply:
Would by nicer with quotes around, I guess

Reply:
True -- let's add the quotes for complete happiness. :)

No need to squash, our bot does that.